src/components/organisms/Forms.js

```diff
@@ -0,0 +1,264 @@
+  spacingUnit,
+  primaryBlue,
+} from '../../utils'
+import { Label, Input, InputDiv } from '../molecules/TextFields'
+
+import { FormButton, SecondaryButton } from '../atoms/buttons'
+import { IconRight } from '../atoms/icons'
+
+const CardContainer = styled.div`
+  width: 100%;
+  max-width: 790px;
+  height: 569px;
+  display: flex;
+  flex-direction: column;
+  background-color: ${white};
+  border-radius: ${radius.sm};
+  box-shadow: 0px 5px 15px rgba(0, 0, 0, 0.1), 0px 4px 6px rgba(0, 0, 0, 0.1);
+  box-sizing: border-box;
+  h4 {
+    font-size: ${typeScale.H4};
+    font-family: ${headerFont};
+    font-weight: ${fontWeight.bold};
+    color: ${neutrals.darkgrey300};
+    margin: 0 auto;
+    padding-top: ${spacingUnit[6]};
+  }
+`
+
+const CardContent = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  height: 100%;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-bottom: ${spacingUnit[2.5]};
+`
+
+const Form = styled.form`
+  max-width: 394px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+`
+
+const FloatDiv = styled.div`
+  display: flex;
+`
+
+const RectDiv = styled.div`
+  width: 2.97px;
+  height: 272px;
+  background: ${primaryBlue[5]};
+  margin-bottom: ${spacingUnit[2]};
+`
+const LoginDiv = styled.div`
+  box-sizing: border-box;
+  padding-bottom: ${spacingUnit[8]};
+  text-align: center;
+
+  p {
+    padding-bottom: ${spacingUnit[4]};
+    font-family: ${headerFont};
+    font-weight: bold;
+    font-size: ${typeScale.para};
+    color: ${neutrals.darkgrey300};
+  }
+  button {
+    font-size: ${typeScale.helperText};
+  }
+  span {
+    padding-left: ${spacingUnit[0]};
+  }
+`
+
+const Para = styled.p`
+  padding: 4px;
+    box-sizing: border-box;
+    background: ${errorCol.error100};
+    border-radius: 2px;
+    color: ${errorCol.error800};
+    margin-top: 0px;
+}
+`
+const Loading = styled.div`
+  display: flex;
+  justify-content: center;
+  height: 100%;
+  align-items: center;
+`
+
+const signupSchema = yup.object().shape({
+  firstname: yup
+    .string()
+    .required('firstname is required')
+    .max(50, 'maximum of 50 characters')
+    .matches(/^[a-zA-Z]+$/i, 'firstname must contain only alphabets')
+    .matches(/^\S{3,}$/, 'firstname cannot contain whitespaces')
+    .trim(),
+  lastname: yup
+    .string()
+    .required('lastname is required')
+    .max(50, 'maximum of 50 characters')
+    .matches(/^[a-zA-Z]+$/i, 'lastname must contain only alphabets')
+    .matches(/^\S{3,}$/, 'lastname cannot contain whitespaces')
+    .trim(),
+  email: yup
+    .string()
+    .email('Please input a valid email')
+    .required('email is required'),
+  password: yup
+    .string()
+    .required('Password is required')
+    .min(5, 'password must be atleast 5 characters')
+    .max(50, 'maximum of 50 characters')
+    .trim(),
+  confirmPassword: yup
+    .string()
+    .required('required')
+    .oneOf([yup.ref('password'), null], 'password does not match'),
```

Comment:
The error message should be `passwords do not match` not `password does not match`